### PR TITLE
Booking controller added

### PR DESCRIPTION
**Pull request by seamuscrawford**
- Added a booking controller - still some bits we'll need to add / amend
- Amended the Car controller index action as it had said Cars.all rather than Car.all
- Added some placeholder views for the booking actions
- Attempted to fill out the first draft of the routes file - think we'll need to adjust this

Let me know if you have any comments!

### CODE REVIEW

File: app/controllers/cars_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class CarsController < ApplicationController
   before_action :set_car, only: [:show]
   def index
-    @cars = Cars.all
+    @cars = Car.all
```
Comment (alexbeet81) [inline app/controllers/cars_controller.rb]: good spot! thank you.

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        @bookings = Booking.all
+    end
+    
+    def new
+        @booking = Booking.new
+    end
+
+    def create
+        @booking = Booking.new(booking_params)
+        @booking.save
```
Comment (alexbeet81) [inline app/controllers/bookings_controller.rb]: change this line to...

```
if @booking.save
      redirect_to booking_path(@booking)
    else
      render :new
    end
```

File: app/controllers/bookings_controller.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    def show; end
+
+    def edit; end
+  
+    def update
+        @booking.update(booking_params)
+        # TODO: add redirect to bookings_path(@booking) when route created
+    end
+  
+    def delete
```
Comment (alexbeet81) [inline app/controllers/bookings_controller.rb]: I think this should change to destroy

Comment (seamuscrawford) [reply app/controllers/bookings_controller.rb]: the name of the action should be 'destroy'? I thought it was 'delete' and then the method called on @booking was 'destroy'? I could be wrong

File: app/controllers/bookings_controller.rb (hunk 2 above)
Comment (seamuscrawford) [reply app/controllers/bookings_controller.rb]: will do thanks!

File: app/controllers/bookings_controller.rb (hunk 3 above)
Comment (alexbeet81) [reply app/controllers/bookings_controller.rb]: Screenshot 2021-06-16 at 13 17 15

I was also unsure. Just checked and it should be destroy.

Comment (seamuscrawford) [reply app/controllers/bookings_controller.rb]: 👍🏻

**Pull request merged**